cot-cli/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(clippy::missing_errors_doc, clippy::missing_panics_doc)]
```

Comment:
Why?

Reply:
Cot CLI doesn't have any public API, so this kind of regime doesn't sound necessary.

Reply:
Fair!